readme.md

```diff
@@ -132,6 +132,68 @@ This is where your stories will be registered and where the Storybook configs wi
 
 This directory will hold your Jest configs and mocks, as well as your [storyshots](https://github.com/storybooks/storybook/tree/master/addons/storyshots) test file. This is a file that contains the snapshots of all your component storybooks.
 
+# About The Stack
+
+## Why this stack?
+
+If you've used Ignite Andross (the first Ignite stack), you know we formerly used Redux for state management, as does much of the community. However, we encountered some pain points with Redux so went in search of a different solution to meet our needs and landed on `mobx-state-tree`. We find that it’s a great middle ground between completely structured (like `redux`) and completely freestyle (like `mobx`). It brings more than just state-management to the table as well (such as dependency injection, serialization, and lifecycle events).
+
+### Some Highlights of MST
+
+MST is...
+
+- Intuitive
+  - With concepts like `props` and `actions`, it feels familiar for React developers
+  - Updating your data means calling functions on objects, rather than dispatching actions.
+  - Feels similar to relational databases, with concepts like `identifiers` (primary keys), `references` (foreign keys), and `views` (calculated fields)
+- Streamlined
+  - No more `actionTypes`, `actionCreators`, or `reducers`
+  - You don't have to declare your usage intentions with `mapStateToProps`, they are inferred
+  - Side-effects are built in, no need for 3rd party libraries like `redux-saga`, `redux-observable`, or `redux-thunk`
+  - Immutability is built-in - no need for `immutable.js` or `seamless-immutable`
+  - `types.compose` and `model.extend` allow for easy code-sharing of common patterns
+- More than state management
+  - Lifecycle hooks like `afterCreate`, `preProcessSnapshot`, and `beforeDestroy` let you have control over your data at various points in it's lifecycle
+  - Dependency injection with `getEnv` allows you to reference your environment (like API or other services)
+- Performant
+  - Round-trip store writes are much faster
+  - Computed values (views) are only calculated when needed
+
+### Downsides
+
+We also recognize no state management solution is perfect. MST has some known downfalls:
+
+- Integration with TypeScript is clunky at times. MST's own typing system is sometimes at odds with what TypeScript wants
+- `mobx` and `mobx-state-tree` both tend to have quite a lot of "magic" or "sorcery" that makes issues hard to debug because you don't always have a clear picture of what's happening
```

Comment:
```suggestion
- `mobx` and `mobx-state-tree` both seem to have "magic" or "sorcery" that makes issues less straightforward to debug because you don't always have a clear picture of what's happening (but using [Reactotron](https://github.com/infinitered/reactotron), which has `mobx-state-tree` support built-in, helps a lot)
```